libdns.go

```diff
@@ -54,7 +59,8 @@ type RecordGetter interface {
 type RecordAppender interface {
 	// AppendRecords creates the requested records in the given zone
 	// and returns the populated records that were created. It never
-	// changes existing records.
+	// changes existing records. Therefore, it is invalid to use this
```

Comment:
Rationale: Hopefully this makes it easier for people to decide whether to use `GetRecords` or `AppendRecords`.